virt/logging_events_monitoring/virt-reviewing-vm-dashboard.adoc

```diff
@@ -0,0 +1,27 @@
+[id="virt-reviewing-vm-dashboard"]
+= Reviewing resource usage by virtual machines
+include::modules/virt-document-attributes.adoc[]
+:context: virt-reviewing-vm-dashboard
+
+toc::[]
+
+Dashboards in the {ProductName} web console provide visual representations of cluster metrics to help you to quickly understand the state of your cluster. Dashboards belong to the xref:../../monitoring/understanding-the-monitoring-stack.adoc#understanding-the-monitoring-stack[monitoring stack] that provides monitoring for core platform components.
+
+The {VirtProductName} dashboard provides data on resource consumption for virtual machines and associated pods. The visualization metrics displayed in the {VirtProductName} dashboard are based on xref:../../virt/logging_events_monitoring/virt-prometheus-queries.adoc#virt-prometheus-queries[Prometheus Query Language (PromQL) queries].
+
+A xref:../../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[monitoring role] is required to monitor user-defined namespaces in the OpenShift Virtualization dashboard.
+
+include::modules/virt-about-reviewing-top-consumers.adoc[leveloffset=+1]
+
+include::modules/virt-reviewing-top-consumers.adoc[leveloffset=+1]
+
+// include::modules/virt-about-reviewing-resource-consumption.adoc[leveloffset=+1]
+
+// include::modules/virt-reviewing-resource-consumption.adoc[leveloffset=+1]
+
+[id="{context}-additional-resources"]
```

Comment:
Technically the repo guidelines suggest a slightly different syntax for "Additional resources" anchors in assemblies:

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies

Reply:
Yep, changed to: [id="additional-resources_reviewing-vm-dashboard"]